src/pages/distroSettings/tabs/HostTab/getFormSchema.tsx

```diff
@@ -53,95 +63,118 @@ export const getFormSchema = ({
             workDir: {
               type: "string" as "string",
               title: "Working Directory",
-            },
-            setupScript: {
-              type: "string" as "string",
-              title: "Setup Script",
+              minLength: 1,
             },
             setupAsSudo: {
               type: "boolean" as "boolean",
               title: "Run script as sudo",
             },
-          },
-        },
-        sshConfig: {
-          type: "object" as "object",
-          title: "SSH Configuration",
-          properties: {
-            user: {
+            setupScript: {
               type: "string" as "string",
-              title: "SSH User",
+              title: "Setup Script",
             },
-            sshKey: {
-              type: "string" as "string",
-              title: "SSH Key",
-              oneOf: sshKeys.map(({ location, name }) => ({
-                type: "string" as "string",
-                title: `${name} – ${location}`,
-                enum: [name],
-              })),
+            userSpawnAllowed: {
+              type: "boolean" as "boolean",
+              title: "Spawnable",
             },
-            authorizedKeysFile: {
-              type: "string" as "string",
-              title: "Authorized Keys File",
+          },
+          dependencies: {
+            arch: {
+              oneOf: [
+                {
+                  properties: {
+                    arch: { enum: windowsArchitectures },
+                    rootDir: {
+                      type: "string" as "string",
+                      title: "Root Directory",
+                    },
+                  },
```

Comment:
Good catch, I moved it to the bottom of the section since that is significantly easier than putting it elsewhere 😅 let me know if it seems reasonable...
<img width="1310" alt="image" src="https://github.com/evergreen-ci/spruce/assets/9298431/676d1741-6d28-485b-b4c5-22c9266f974b">